modules/react-components/src/components/input/url-input.tsx

```diff
@@ -328,7 +328,7 @@ export const URLInput: FunctionComponent<URLInputPropsInterface> = (
             return (
                 <LabelWithPopup
                     className="cors-details-popup"
-                    popupHeader={ t("devPortal:components.URLInput.withLabel.positive.header") }
+                    popupHeader={ t("console:develop.features.URLInput.withLabel.positive.header") }
```

Comment:
React Components module should not depend on the locale bundles of the app. Ideally, URLInput component should have a default english strings as `defaultProps` and users should be able to pass the translated string to the component. Shall we add a TODO and a link to an issue to get this fixed. Need to do the same for `TransferList` component as well.

Reply:
fixed with commit 76d6c86. This issue will track the mentioned issue. https://github.com/wso2/product-is/issues/10693